components/error-summary/src/test.tsx

```diff
@@ -1,86 +1,52 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { mount } from 'enzyme';
+import { fireEvent, render } from '@testing-library/react';
 
 import ErrorSummary from '.';
-import ErrorSummaryExample, { heading, description, errors } from './fixtures';
 
 describe('error summary', () => {
-  const wrapperErrorSummary = mount(<ErrorSummaryExample />);
-
-  it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<ErrorSummaryExample />, div);
+  it('should render without crashing', () => {
```

Comment:
I think tests like this are giving false code coverage metrics and should be removed or updated to assert key features of a default render

Reply:
https://github.com/govuk-react/govuk-react/issues/910

Reply:
makes sense, and I'd already been thinking roughly along those lines (I just hadn't quite got that far :grin:) - several of these tests are now checks that they essentially pass thru children which, for most of them, is their default behaviour

with that in mind, given the rest of the coverage in this file, this particular test can be deleted

Reply:
agreed please either:
- delete tests like this rather than migrate
- reword them and add assertions to cover effects of default props

Reply:
all the tests that have been edited so far have either had their "should render without crashing" test deleted, replaced with a "renders contents without crashing" that asserts that children passed to the component get rendered, or replaced with other tests for default props.

for now, or CI hasn't been updated to use the merged coverage - I wasn't quite sure how to adjust that.  so far tho coverage has not dropped for unit tests

there are some components with updated tests that have default props where these things are not explicitly tested in unit tests - these are all, I believe, setting a default `size` props to give a default font size.  I'm inclined to think of that as an implementation detail, and best checked via storybook.  an explicit test for "this is size XLARGE by default" feels wrong - and a test of "has a default size" also feels not quite right

Reply:
> setting a default size props to give a default font size

Yes, I'd put it like this:

Any props (or similar forks) that, as far as the user is concerned, only effects the visual output (such as font size and spacing), should be tested via visual regression and we shouldn't be concerned about coverage via unit tests. The important thing to note is there should be unit tests where there is a significant accessibility functionality such as 'labeled by' as this is not visual-only.